ome/omero-figure
# Fix JCB -> omero.lifesci.dundee.ac.uk links in demo 1.json

See https://github.com/ome/omero-figure/pull/326 - Ported fixes on that PR to the ```gh-pages-staging``` branch as required to release as described at https://docs.openmicroscopy.org/contributing/ci-docs.html#term-figure-help-staging

To test:
 - See https://figure.staging.openmicroscopy.org/demo/#file/1
 - Figure should display as normal (images loading etc) and can be edited (change rendering settings etc)
